src/Keboola/DbExtractor/Extractor/Common.php

```diff
@@ -257,7 +279,7 @@ public function getTables(array $tables = null)
     }
 
 
-    private function quote($obj)
+    private function quote(string $obj): string
     {
         return "`{$obj}`";
```

Comment:
Not relevant here, just curious, but is this really a way to quote? What if `$obj` contains a backtick?

Reply:
mysql identifiers can't contain backtick afaik.

Reply:
But those are user-supplied, aren't they? So what stops user from using `table[backtick]name` as tablename [here](https://github.com/keboola/db-extractor-common/pull/44/files/94449f896700d2a938599f0c7f8f2e0c5800e935#diff-ae8599dc51a981b5f2229546d3f4fc3dL120)?

Reply:
I suppose that the user could (via the api) create a config with such a table name, but it will turn into a `UserException` when the `SimpleQuery` is executed, so imho that's expected behaviour.

Reply:
More reasonable "tablename" would be something along the lines of
```
id`, users.* JOIN users ON 1=1; --
```
But I guess user is accessing their own DB instance, so not much more damage can be done this way that couldn't be made in other ways...

Reply:
Yeah, if they use the Advanced query, we attempt to execute anything that they write there :-D.  So it's very `At your own risk`.  (the quote method isn't used in that case fwiw).

Reply:
Oh, of course, I've already seen that when Zendesking. Should have known better, sry ;)